package.json

```diff
@@ -34,6 +34,9 @@
     "joi": "^14.3.1",
     "yargs": "^11.0.0"
   },
+  "engines": {
+    "node": "^8.3.0 || ^10.16"
```

Comment:
Is there any reason to specify the minor versions explicitly instead of just `^8.0.0` and `10.0.0`?

Reply:
I don't have any specific reason for requiring a specific minor version so happy to change this.